packages/client/mutations/toasts/popNotificationToast.ts

```diff
@@ -50,24 +53,20 @@ export const popNotificationToastOnNext: OnNextHandler<
     return
   }
 
-  if (notificationSnack.action) {
```

Comment:
-1 Why did you remove the action part here? Doesn't that break existing notifications?

Reply:
@Dschoordsch this part is not needed anymore. The previous code marked the notification as read only if we pressed the action button. That's why we checked if button exists.

But it could be a snackbar without any button. Or user could just click on other parts of the snackbar to close it.

So now I handle these cases by marking notification as read if snackbar was manually dismissed.

This is the same logic we follow for notifications in header, if you click on it it will be marked as clicked. 

cc @jmtaber129